views/js/previewer/plugins/content/scoringHighlighter.js

```diff
@@ -108,9 +107,9 @@ define([
             function getAllRanges(selection) {
                 const allRanges = [];
 
-                for (let i = 0; i < selection.rangeCount; i++) {
-                    allRanges.push(selection.getRangeAt(i));
-                }
+                    for (let i = 0; i < selection.rangeCount; i++) {
```

Comment:
extra indent :thinking:

Reply:
oh, I forgot to run prettier for that project 😔